sys-libs/pam/metadata.xml

```diff
@@ -12,6 +12,10 @@
 		flag will create a PAM module that links to the Berkeley DB (as
 		provided by <pkg>sys-libs/db</pkg>) installed in /usr/lib and
 		will thus not work for boot-critical services authentication.
+	</flag>
+	<flag name="xcrypt">
```

Comment:
Could you please fix the CI failure (rebasing may do it, actually, given we've just unmasked for the general libxcrypt migration) and also add this to the package.use.force for libxcrypt we have in profiles (it's there for USE=system already, just add this on a new line)?